lib/player.js

```diff
@@ -1791,8 +1791,28 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       // Make sure that the app knows of the new buffering state.
       this.updateBufferState_();
 
+      const bufferRange = () => {
```

Comment:
Is this because the buffered range can extend beyond the seek range?  Please add a brief comment here.

Reply:
No, this is to simply use the buffered range instead of the seek range. For a VOD, the buffered range is smaller than the seek range, and this means we store less data.